cli/cli.go

```diff
@@ -29,24 +31,41 @@ type LoggingOpts struct {
 }
 
 // AdminOpts is a re-export of the admin type so servers don't need to import it directly.
-type AdminOpts = admin.Opts
+type AdminOpts struct {
+	admin.Opts
```

Comment:
I dunno what the linter is moaning about here. Should be fine.

Reply:
Yeah, going to attempt to explicitly import as `admin`